pipeline/scraper/test.py

```diff
@@ -1,16 +1,115 @@
-# This is used to test PSQL, ignore this
-#!/usr/bin/python
-import psycopg2
+# Test Script
+# To run test, use the command under the main folder
+# DB_USER=postgres DB_PASSWORD=secret DB_HOST=localhost TOKEN=<token> python3 -m unittest scraper/test.py -v in the scraper folder
+
 import sys
+sys.path.append('./scraper')
+
 import PSQL
- 
-def main():
-    # db = PSQL.connectToDB()
-    # cur = db.cursor()
-    # cur.execute("SELECT * FROM applications;")
-    # result = cur.fetchall()
-    # print(result)
-    print(type({}))
+import GitHubQuery
+import unittest
+import json
+import urllib
+import datetime
+
+def make_orderer():
+    order = {}
+
+    def ordered(f):
+        order[f.__name__] = len(order)
+        return f
+
+    def compare(a, b):
+        return [1, -1][order[a] < order[b]]
+
+    return ordered, compare
+
+ordered, compare = make_orderer()
+unittest.defaultTestLoader.sortTestMethodsUsing = compare
+
+class TestMyClass(unittest.TestCase):
+
+    @ordered
+    def test_runQuery(self):
```

Comment:
This method doesn't seem to serve any purpose?

Reply:
deleted